tools/monitoring/mysql/db_mysql.go

```diff
@@ -71,3 +75,42 @@ func (c DBConfig) dataStoreName(dbName string) string {
 
 	return fmt.Sprintf("%sunix(%s)/%s", cred, "/cloudsql/"+c.Instance, dbName)
 }
+
+// PubsubMsgHandler adds record(s) to ErrorLogs table in database,
+// after parsing build log and compares the result with config yaml
+func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumber int) error {
+	config, err := config.ParseYaml(configURL)
+	if err != nil {
+		return err
+	}
+
+	logs, err := logs.ParseLog(buildLogURL, config.CollectErrorPatterns())
```

Comment:
```suggestion
	pattern, err := logs.ParseLog(buildLogURL, config.CollectErrorPatterns())
```

Reply:
this is a list of ErrorLog structs, not just patterns.